Api/Report/ReportInterface.php

```diff
@@ -13,4 +13,5 @@
      * @return ReportRequest
      */
     public function getRequest(array $columns, $timePeriod);
+
```

Comment:
this empty line should not be there, please check the PSR code style rules and set phpstorm to use them -> preferences -> code style -> php -> set from -> PSR